### DISCUSSION TITLE
Ubuntu runner prematurely shutdown

### DISCUSSION THREAD

**Discussion by petersteneteg** [community/community#26830] [Actions]
<p>I run a matrix job with fail-fast = false, and Ubuntu-20.04 runner with always gets shutdown for now obvious reason, I only get this message in the log after the build has been terminated:</p>
<blockquote>
<p>The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.</p>
</blockquote>
<p>Link to job<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/inviwo/inviwo/runs/1176155361?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/3638222/181100223-41d31cd9-2198-41fa-8838-4fa66435a7db.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d5144243c16bd6ddb174ca12fe66c086f7c49dd3_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d5144243c16bd6ddb174ca12fe66c086f7c49dd3_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d5144243c16bd6ddb174ca12fe66c086f7c49dd3.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d5144243c16bd6ddb174ca12fe66c086f7c49dd3_2_10x10.png"></div>

<h3><a href="https://github.com/inviwo/inviwo/runs/1176155361?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">inviwo/inviwo</a></h3>

<p>Inviwo - Interactive Visualization Workshop. Contribute to inviwo/inviwo development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/petersteneteg">@petersteneteg</a>,</p>
<p>Did you try other GitHub-hosted Ubuntu runners (<strong>Ubuntu 18.04</strong> and <strong>Ubuntu 16.04</strong>)? Did they work?<br>
Please have a try with them, and any update, feel free to tell us.</p>
<p>In addition, I have created an issue ticket (<a href="https://github.com/actions/runner/issues/724" rel="noopener nofollow ugc">actions/runner#724</a>) to help you report the problem to the appropriate engineering team for further investigation and evaluation.</p>
<p>You can follow this issue ticket and add your comments to it.</p>

**Comment by petersteneteg** [comment, marked as answer]
<p>I switch from using make to ninja to build then the error disappeared</p>

**Comment by petersteneteg** [comment]
<p>I can not build with ubuntu 18.04 or older, the c++ compiler is to old</p>
